### ISSUE TITLE
Consider adding a 'cite this' link on OLS

### ISSUE THREAD

**Issue by cmungall** [EBISPOT/OLS#61]
cc @mellybelly - this would be the same as the URI, or an accepted shortform

Also it can be useful to have a copy button that copies the URL into your copy buffer/clipboard. Saves 0.1-0.2s of mouse activity

**Comment by jmcmurry** [comment]
+1

**Comment by LLTommy** [comment]
Obviously, this could be easly done. I will talk to @simonjupp if/how he wants this to be implemented.

**Comment by LLTommy** [comment]
We will introduce a (actually two) 'copy this' button in the next version of OLS. We still think about the format of the 'cite this' button - this one should not only be the URI but a proper citation of the OLS. However, most likley we will also introduce the cite this button as suggested here.
